language/message/de.po

```diff
@@ -1,2247 +1,1646 @@
 msgstr[1] "{subclass} mit %d Sternen"
 
-#: public/build/javascript/bundle-30a43b5139568d791e2c66fd2266bb81.js:109698
 msgid "Starred {subclass}"
 msgstr "Markiert {subclass}"
 
-#: public/build/javascript/bundle-30a43b5139568d791e2c66fd2266bb81.js:109887
-msgid "Reviews"
-msgstr "Bewertungen"
-
-#: public/build/javascript/bundle-30a43b5139568d791e2c66fd2266bb81.js:109908
-msgid "TripAdvisor user"
-msgstr "TripAdvisor-Nutzer"
-
-#: public/build/javascript/bundle-30a43b5139568d791e2c66fd2266bb81.js:109919
-msgid "Read the full review"
-msgstr "Komplette Bewertung lesen"
-
-#: public/build/javascript/bundle-30a43b5139568d791e2c66fd2266bb81.js:109928
-msgid "Read more reviews on TripAdvisor"
-msgstr "Weitere Bewertungen bei TripAdvisor lesen"
-
-#: public/build/javascript/bundle-30a43b5139568d791e2c66fd2266bb81.js:109993
-msgid "website"
-msgstr "Website"
-
-#: public/build/javascript/bundle-30a43b5139568d791e2c66fd2266bb81.js:110099
 msgid "See more nearby services"
 msgstr "Weitere Angebote in der Nähe anzeigen"
 
-#: public/build/javascript/bundle-30a43b5139568d791e2c66fd2266bb81.js:110102
 msgid "Reduce"
 msgstr "Weniger anzeigen"
 
-#: public/build/javascript/bundle-30a43b5139568d791e2c66fd2266bb81.js:110140
 msgid "Unfold to see quick actions"
 msgstr "Aufklappen, um die Schnell-Aktionen zu sehen"
 
-#: public/build/javascript/bundle-30a43b5139568d791e2c66fd2266bb81.js:110192
+msgid "by car"
+msgstr "mit dem Auto"
+
+msgid "transit"
+msgstr "Linienverkehr"
+
+msgid "on foot"
+msgstr "zu Fuß"
+
+msgid "by bike"
+msgstr "mit dem Rad"
+
 msgid "Services nearby"
 msgstr "Angebote in der Nähe"
 
-#: public/build/javascript/map-6a915ed20e2690ae1c9b.bundle.js:2545
-#: src/mapbox/locale.js:4
 msgid "My location"
 msgstr "Mein Standort"
+
+#~ msgid "Products for everyday life."
+#~ msgstr "Produkte für das tägliche Leben."
+
+#~ msgid "   Search   "
+#~ msgstr "  Search"
+
+#~ msgid "Responsible search that respects your privacy."
+#~ msgstr "Verantwortungsvolle Suche, die deine Privatsphäre respektiert."
+
+#~ msgid "Open Search"
+#~ msgstr "Search öffnen"
+
+#~ msgid "Maps"
+#~ msgstr "Maps"
+
+#~ msgid "The map that does not track you."
+#~ msgstr "Die Karte, die dich nicht verfolgt."
+
+#~ msgid "Open Maps"
+#~ msgstr "Maps öffnen"
+
+#~ msgid "Junior"
+#~ msgstr "Junior"
+
+#~ msgid "Responsible search adapted to 6-12 year olds."
+#~ msgstr "Verantwortungsvolle Suche an 6-12-Jährige angepasst."
+
+#~ msgid "Open Junior"
+#~ msgstr "Junior öffnen"
+
+#~ msgid "We believe in an alternate model"
+#~ msgstr "Wir glauben an ein alternatives Modell"
+
+#~ msgid "See the usual opening hours"
+#~ msgstr "Die üblichen Öffnungszeiten anzeigen"
+
+#~ msgid "Reviews"
+#~ msgstr "Bewertungen"
+
+#~ msgid "TripAdvisor user"
+#~ msgstr "TripAdvisor-Nutzer"
+
+#~ msgid "Read the full review"
+#~ msgstr "Komplette Bewertung lesen"
+
+#~ msgid "Read more reviews on TripAdvisor"
+#~ msgstr "Weitere Bewertungen bei TripAdvisor lesen"
```

Comment:
Not sure why these ones got commented?

Feels like it will probably need more extensive testing before we merge this :sob: